src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Components/Constants.cs

```diff
@@ -63,7 +63,7 @@ public static class Constants
         /// <summary>
         /// The default policy API version.
         /// </summary>
-        public static readonly string PolicyApiVersion = "2016-04-01";
+        public static readonly string PolicyApiVersion = "2016-12-01";
```

Comment:
just curious as to why the new API version date is set to December instead of a more recent date?